src/gzip.rs

```diff
@@ -1199,10 +1170,7 @@ where
 mod tests {
     use super::*;
     use crate::finish::AutoFinish;
-    #[cfg(feature = "no_std")]
     use core2::io::{Read, Write};
```

Comment:
```suggestion
    use core2::io::{Read, Write};
    #[cfg(not(feature = "std"))]
    use alloc::vec;
```

Reply:
This is not resolved. You can remove the conditional compilation, but you still need the `use alloc::vec` import.

Reply:
Correct, with the same communication mixup as above, done.